lighthouse-core/formatters/partials/critical-request-chains.html

```diff
@@ -93,13 +135,14 @@
         - <span class="cnc-node__chain-duration">{{chainDuration startTime this.node.request.endTime }}ms, {{formatTransferSize this.transferSize}}KB</span>
       {{/unless}}
     </span>
+
+    {{#each this.node.children as |child| }}
+      {{#createContextFor ../node.children @key ../treeMarkers ../isLastChild ../startTime ../transferSize }}
+        {{> writeNode this }}
+      {{/createContextFor }}
+    {{/each}}
```

Comment:
Make the request tree more like a tree in DOM. So we can have a better representation of requests in DOM and make it easier to show correct toggle icon.